src/tests/java/htsjdk/samtools/CramFileWriterTest.java

```diff
@@ -231,4 +231,24 @@ public void test_roundtrip_tlen_preserved() throws IOException {
         Assert.assertEquals(records.size(), i);
     }
 
+    @Test
+    public void testCRAMQuerySort() throws IOException {
+        final File input = new File("testdata/htsjdk/samtools/count_reads.cram");
+        final File reference = new File("testdata/htsjdk/samtools/count_reads.fasta");
+        final File outputFile = File.createTempFile("tmp.", ".cram");
+
+        try (final SamReader reader = SamReaderFactory.makeDefault().referenceSequence(reference).open(input);
+             final SAMFileWriter writer = new SAMFileWriterFactory().makeWriter(reader.getFileHeader().clone(), false, outputFile, reference)) {
+            for (SAMRecord rec : reader) {
+                    writer.addAlignment(rec);
+            }
+        }
+
+        try (final SamReader outReader = SamReaderFactory.makeDefault().referenceSequence(reference).open(outputFile)) {
+            for (final SAMRecord rec : outReader) {
+            }
+        }
+
+    }
+
```

Comment:
This looks like the test case for writing and then reading back in a query-sorted cram. Now that it doesn't throw it really should have an assert to verify the result is correct. Also we please rename the test file to something relevant to this context, maybe "querysorted.cram".